src/configure/configurers/AksAzureResourceSelector.ts

```diff
@@ -0,0 +1,12 @@
+import { GenericResource } from "azure-arm-resource/lib/resource/models";
+import { TargetResourceType } from "../model/models";
+import { IAzureResourceSelector } from "./IAzureResourceSelector";
+
+export class AksAzureResourceSelector implements IAzureResourceSelector {
+     async getAzureResource(inputs): Promise<GenericResource> {
+          return { type: TargetResourceType.AKS };
```

Comment:
I did not understand this. Should we not be showing list of AKS resources to user?

Reply:
For AKS, everything is done in a separate function called setParameters which we have reused as it is.  AKS resources dropdown is shown to the user while processing the predefined AKS template.